### ISSUE TITLE
mobile/desktop display

### ISSUE THREAD

**Issue by rimmesbe** [nvcanales/CapStone#4]
You have a lot of displays toggling to keep track of. You might want to just have 1 mobile class and 1 desktop class that you put on every element you need to toggle. This way you can just change the display setting in 1 place.

https://github.com/nvcanales/CapStone/blob/master/resources/css/style.css#L573-L581

**Comment by nvcanales** [comment]
Yes after I had it all working and submitted I thought why don't I simply but it was already to late. 
Thank you for the reminder.
